kimchi/src/circuits/constraints.rs

```diff
@@ -413,7 +413,7 @@ impl<F: FftField + SquareRootField> LookupConstraintSystem<F> {
                 // table 0.
                 let dummy_lookup = JointLookup {
                     entry: vec![],
-                    table_id: 0,
+                    table_id: F::zero(),
```

Comment:
the inconvenience of havint a `table_id: F` at this point is that we can't specify these things with an enum or a hardcoded `XOR_TABLE_ID` or something

Reply:
As above: this uses the unwritten assumption that table 0 is either empty or has an all-zero entry

Reply:
Perhaps that assumption ought to be written

Reply:
or better: asserted